gitlab/config.py

```diff
@@ -61,11 +61,19 @@ def __init__(self, gitlab_id=None, config_files=None):
         self.ssl_verify = True
         try:
             self.ssl_verify = self._config.getboolean('global', 'ssl_verify')
+        except ValueError:
+            # Value Error means the option exists but isn't a boolean.
+            # Get as a string instead as it should then be a local path to a
+            # CA bundle.
+            self.ssl_verify = self._config.get(self.gitlab_id,
+                                               'ssl_verify')
         except Exception:
             pass
         try:
             self.ssl_verify = self._config.getboolean(self.gitlab_id,
                                                       'ssl_verify')
+        except ValueError:
+            self.ssl_verify = self._config.get(self.gitlab_id, 'ssl_verify')
```

Comment:
I think it's worth adding an explanation comment here as well, just in case someone is following a stack trace they may not see the one above.